### DISCUSSION TITLE
app not working

### DISCUSSION THREAD

**Discussion by ihabmzaky** [Budibase/budibase#7872] [Help] [answered]
My app was working fine until yesterday both online and offline.  Updated to new version today and got blank screens in the design view.  My published app shows same view, screen slot!  None of the views are visible anymore.  see attached.  HELP please urgently.
![image](https://user-images.githubusercontent.com/109164523/191491017-4da28cdb-2685-4384-9e76-7e529c67af39.png)

**Comment by joebudi** [comment, marked as answer]
Hey. Try clicking 'update available ' - that might work.

![Screenshot_20220921-125157.jpg](https://user-images.githubusercontent.com/49767913/191497015-bbf1ac41-c11a-427f-8337-e19a5f2a5ec1.jpg)

**Reply by ihabmzaky** [reply]
I just figured it out.  I had to "detach" screen from the right toolbar. Strange behavior but all working now.  Thanks
